package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "turbo run dev",
     "lint": "turbo run lint",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
-    "publish-packages": "turbo run build lint test && changeset version && changeset publish"
+    "release": "turbo run build lint && changeset publish"
```

Comment:
for release, we can just have  `changeset publish` . the run and build are done on every pr before merge so no need to do it here again.

Reply:
Updated.